### DISCUSSION TITLE
I have accessed the API and there is a transaction HASH, but I can't query it on the chain, what should I do?

### DISCUSSION THREAD

**Discussion by Sun-dong** [filecoin-project/lotus#8252] [General Q&A]
I only encountered it after upgrading to 1.15.3 rc, how can I solve this problem? What should I do

**Comment by Sun-dong** [comment]
for example : bafy2bzaceblsp7cemqhhq3ag76piektj3zfobbepurrm6sgtkqeln2blghqkm

lotus chain getmessage bafy2bzaceblsp7cemqhhq3ag76piektj3zfobbepurrm6sgtkqeln2blghqkm
{
  "Message": {
    "Version": 0,
    "To": "f1broppsxgbanx622adgvbnanwub2dqwhywy7lora",
    "From": "f1l7tf2i7uq6nj6aqjyhkre4hegnzntdzbenlacza",
    "Nonce": 1246,
    "Value": "1168567780000000000",
    "GasLimit": 608335,
    "GasFeeCap": "4823380768",
    "GasPremium": "100147",
    "Method": 0,
    "Params": null,
    "CID": {
      "/": "bafy2bzacea5tq3sqcr2pzirjcghriastgq4k7qbwvfw5sprjafzr6wjw3mus2"
    }
  },
  "Signature": {
    "Type": 1,
    "Data": "jNdv9HMYm1JQ+r9qIeR7eDz2jOROkjMUyzanBIzF2ehPA8PHZOWb07ydCdd3Z/OxrQq5ktQ6uGdUUrqTduY0OQE="
  },
  "CID": {
    "/": "bafy2bzaceblsp7cemqhhq3ag76piektj3zfobbepurrm6sgtkqeln2blghqkm"
  }
}

**Comment by LexLuthr** [comment]
@Sun-dong Can you explain a bit more what "transaction HASH" you are not able to query? What is the error you get? Where are you running this query?
From your current description, I am assuming you are not able to query a message using the message CID. If that is the case, then maybe the node where you are querying it does not have the block containing the message. It happens usually because this message might be older and the node simply does not have the older data.

**Comment by LexLuthr** [comment]
I am archiving this discussion as there has been no new update from the author for more than 7 days.
